api/tests/core/search/test_serialize_algolia.py

```diff
@@ -216,6 +220,19 @@ def test_serialize_offer_thumb_url():
     assert serialized["offer"]["thumbUrl"] == f"/storage/thumbs/products/{humanize(offer.productId)}"
 
 
+def test_serialize_offer_gtl():
+    product = offers_factories.ProductFactory(extraData={"gtl": "01030100"})
```

Comment:
... ça répond au fil de discussion plus haut : ton test est incorrect, la bonne clé est `gtl_id` : 

```suggestion
    product = offers_factories.ProductFactory(extraData={"gtl_id": "01030100"})
```